recipes/glew/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "glew" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "cc88d2f70a185185eb050eb13789f498425fcd58230bfd65da496b8008f046f2" %}
+{% set dev_url = "https://github.com/nigels-com/glew" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: {{ dev_url }}/releases/download/{{ name }}-{{ version }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+  patches:
+    - CMakeLists.patch  # [win and not py35]
+  
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python  # [win]
```

Comment:
The example recipe says:

```yaml
# if your project compiles code (such as a C extension) then add `toolchain` as a build requirement.
```

You can check the [toolchain feedstock activate script](https://github.com/conda-forge/toolchain-feedstock/blob/master/recipe/activate.sh) to see what it is doing exactly.
In a nutshell, it prepares your environment correctly on windows, osx and linux to be able to compile things correctly.